x-pack/legacy/plugins/siem/public/pages/detection_engine/rules/edit/translations.ts

```diff
@@ -28,3 +28,9 @@ export const SORRY_ERRORS = i18n.translate(
 export const BACK_TO = i18n.translate('xpack.siem.detectionEngine.editRule.backToDescription', {
   defaultMessage: 'Back to',
 });
+
+export const SUCCESSFULLY_SAVED_RULE = (ruleName: string) =>
+  i18n.translate('xpack.siem.detectionEngine.rules.update.successfullyRuleSavedTitle', {
```

Comment:
nit

```suggestion
  i18n.translate('xpack.siem.detectionEngine.rules.update.successfullySavedRuleTitle', {
```

Reply:
Also, may want to have a case for `emptystring` that says `Rule was saved` as you send `emptystring` if there is no name which would result in a toast saying ` was saved`:

https://github.com/elastic/kibana/blob/a7c5dbfc7fd84557e504eadf227755e8818f9115/x-pack/legacy/plugins/siem/public/pages/detection_engine/rules/edit/index.tsx#L276